.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -69,6 +76,26 @@ on:
         description: >
           ID of the Application Insights resource that the release annotation for the swap should be added to.
           This can e.g. be looked up on the Azure Portal under the given AI resource's Overview page -> JSON View.
+      swap-code-repo:
```

Comment:
More like `app-code-repo`. This sounds to me like the repo of the swap operation's code, what's the current consumer repo.